### PR TITLE
Create 23editorcs.dat

### PR DESCRIPTION
**Pull request by quynguyen2303**
Nguyễn Phú Quý registered to VISC.

### CODE REVIEW

File: reg/23editorcs.dat [hunk 1]
```diff
@@ -0,0 +1,2 @@
+SHA-256:(SHA-256(VISC:23editorcs))
```
Comment (vietlq) [inline reg/23editorcs.dat]: Please look at the example for my string `VISC:vietlq`:

```
SHA-256:815f421ea518866d186845a4be3a46828ed21429a896654b36b7ddba1e464d04
KECCAK-256:47620e53f94f7d9f4560e4f89e25bb60513941f39fbc30d2494333431dc6c234
```

You need to hash your string too :)

**Review by vietlq — CHANGES_REQUESTED** [review]
You need to hash "VISC:23editorcs" using SHA-256 and KECCAK-256.

**Comment by quynguyen2303** [comment]
Yes, I already updated on the last one.

**Comment by vietlq** [comment]
@23editorcs still isn't right. Show me your code how did you derive the hashes.

**Comment by quynguyen2303** [comment]
I actually used your code to derive the hashes since it printed the output. My code:
python3 verify-sig.py 23editorcs
Is it the right way to do?

**Comment by quynguyen2303** [comment]
@vietlq  Sorry, maybe I typed a wrong string last time. I just updated it again. Please, check it. Thanks

**Review by vietlq — APPROVED** [review]

**Comment by vietlq** [comment]
@23editorcs thanks for your interest. Please send a request to https://www.facebook.com/groups/VISCHub/ and I will add you :)

**Comment by quynguyen2303** [comment]
Thank for your patience with me :). I have sent the request.

**Pull request merged**